.circleci/circlescript.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
Functions could complicate things especially when it comes to environment variables and `exit`, I have run into issues before where the shell got a bit confused about what is going on. I wanted to prevent that.